build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -0,0 +1,147 @@
+      name: NSXT Project
+      type: string
+    - description: PublicIPv4Block assigned to the Namespace
+      jsonPath: .spec.PublicIPv4Block
+      name: PublicIPv4Block
+      type: string
+    - description: PrivateCIDRs assigned to the Namespace
+      jsonPath: .spec.PrivateCIDRs
+      name: Private CIDRs
+      type: string
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: VPCNetworkConfiguration is the Schema for the vpcnetworkconfigurations
+          API.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: VPCNetworkConfigurationSpec defines the desired state of
+              VPCNetworkConfiguration. There is a default VPCNetworkConfiguration
+              that applies to Namespaces do not have a VPCNetworkConfiguration assigned.
+              When a field is not set in a Namespace's VPCNetworkConfiguration, the
+              Namespace will use the value in the default VPCNetworkConfiguration.
+            properties:
+              appliedToNamespaces:
+                description: Namespaces to be applied. The default VPCNetworkConfiguration
+                  will not have AppliedToNamespaces set.
+                items:
+                  type: string
+                type: array
+              defaultSubnetPrefixLength:
+                default: 26
+                description: Default subnet prefix length for Subnet. Defaults to
+                  26.
+                type: integer
+              defaultSubnetType:
+                default: public
+                description: DefaultSubnetType defines the type of the default SubnetSet
+                  for PodVM and VM. Must be public or private. Defaults to public.
+                enum:
+                - public
+                - private
+                type: string
+              edgeCluster:
+                description: Edge cluster path on which the networking elements will
+                  be created.
+                type: string
+              nsxtProject:
+                description: NSX-T Project the Namespace associated with.
+                type: string
+              privateCIDRs:
```

Comment:
Do we need to call it privateIPv4CIDRs instead? As the public IP block has v4 and v6 parameters

Reply:
sounds good. Thanks Danting. changed to privateIPv4CIDRs.